context/context.go

```diff
@@ -287,7 +288,12 @@ func (context *AptlyContext) _database() (database.Storage, error) {
 	if context.database == nil {
 		var err error
 
-		context.database, err = goleveldb.NewDB(context.dbPath())
+		if context.config().DatabaseBackend.Type == "etcd" {
+			context.database, err = etcddb.NewDB(context.config().DatabaseBackend.Url)
+		} else {
+			context.database, err = goleveldb.NewDB(context.dbPath())
```

Comment:
I see the reason for using `DatabaseBackend.Type` as recommended. I wonder though if we can adapt `dbPath` (i.e. `rootPath/db`) into your new `DbConfig` structure? We must be backward compatible though. Maybe it's worth raising a separate ticket once this is merged.

Reply:
I don't quite understand. At present, this dbpath should be available only for leveldb, and the current processing method should be compatible with it. My idea is that if the dbconfig configuration in the configuration file is empty, the default leveldb is used as the back-end, and leveldb compatibility should not be required in dbconfig. This is also explained in the man document.